docs/readme-ciscojabber.md

```diff
@@ -0,0 +1,344 @@
+# BotKit Starter for Cisco Jabber  #
+Botkit is designed to ease the process of designing and running useful, creative bots that live inside Cisco Jabber.
+Botkit features a comprehensive set of tools to deal with Cisco Jabber, and allows developers to build interactive bots and applications that send and receive messages just like real humans.
+This document covers the Cisco Jabber-specific implementation details only.
+
+## Getting Started ##
+- Install Botkit
+- Ask admin to create a Jabber user for the bot in either Cisco IM&Presence Server (on-premise deployment) or Cisco Webex Messenger (cloud deployment), then get the jid and password from the admin.
+Jabber bots can send and receive messages, and in many cases, appear alongside their human counterparts as normal Jabber users.
+
+## Working with Cisco Jabber ##
+Cisco Jabber bot uses node-xmpp to connect to Cisco Unified IM & Presence server or Cisco WebEx Messenger and can only use the password authentication methods. The bot is relying on starndard XMPP protocol to send/receive messages. You can refer to [RFC6120](https://tools.ietf.org/html/rfc6120) and [RFC6121](https://tools.ietf.org/html/rfc6121) to learn more about how to connect to XMPP server and how to create the XMPP message stanza. 
+
+Cisco Jabber bot can take part in 1-1 chat or group chat conversations and can provide information or other services for the conversations. It’s better to use persist store for the Botkit controller, otherwise when restart the bot, it will lose the group information and cannot received the message from a group chat.
+
+The full code for a simple Cisco Jabber bot is as below and you can find it in ../examples/jabber_bot.js:
+
+~~~ javascript
+    const Botkit = require('./lib/JabberBot.js');
+    var controller = Botkit({
+    json_file_store: './bot_store/'
+    });
+    var bot = controller.spawn({
+    client: {
+        jid: ‘xx@domain.com',
+        password: *,
+        host: "hostname.domain.com",
+        port: 5222
+        }
+    });
+    controller.hears(['hello'], ['direct_mention', 'direct_message'], function (bot, message) {
+        bot.reply(message, 'Hi');
+    });
+    controller.on('direct_mention', function (bot, message) {
+        bot.reply(message, 'You mentioned me in a group and said, "' + message.text + '"');
+    });
+    controller.on('direct_message', function (bot, message) {
+        bot.reply(message, 'I got your direct message. You said, "' + message.text + '"');
+    });
+~~~
+    
+## Bot Options ##
+When spawn bot from the Botkit controller, there are several options available.
+
+| Argument | Description
+|--- |---
+| jid | Jid of the jabber bot 
+| password | Password of the jabber bot
+| host | host of the Cisco Unified IM & Presence server, no need for bot of Cisco WebEx Messenger
```

Comment:
If all descriptions start with capital letter, this one should start with `H`.
`not necessary` would look better than `no need`.